### ISSUE TITLE
Unnecessary kindKey Field in CrFwInRepKindDesc_t Structure

### ISSUE THREAD

**Issue by pasetti** [pnp-software/cordetfw#75]
As far as I can tell, filed kindKey in the CrFwInRepKindDesc_t Structure is unused and unnecessary (the "kind keys" for incoming reports are held in array inRepKindKey in the InFactory module). This field is currently initialized to zero in the InFactoryUserPar.

I need to double-check my findings and then I should remove the unnecessary field.

**Comment by pasetti** [comment]
I have removed field kindKey in the CrFwInRepKindDesc_t Structure in CrFwConstants.h. I have also updated the initialization of this data structure in the InFactoryUserPar in the TestSuite. 

I have checked the InFactoryUserPar files in the three applications in cordetfw-examples but they did not include the definition of this parameter.

The only remaining change is to remove kindKey from the generator of the CORDET Editor. The change must be done in function gen_cr_fw_in_factory_user_par of file gen_cfw.py.
